task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -58,8 +62,9 @@ spec:
         image_with_digest="$(params.IMAGE_URL)@$(params.IMAGE_DIGEST)"
 
         if ! oc image extract "${image_with_digest}" ; then
-          echo "Unable to extract image! Skipping checking binaries!"
-          HACBS_TEST_OUTPUT="$(make_result_json -r ERROR -t 'Unable to extract image! Skipping checking binaries!')"
+          echo "Unable to extract image, unable to validate the extracted binaries of the image!"
+          note="Task \`$(context.task.name)\` failed: Failed to extract image by command \`oc extract\` so that it is unable to validate the exracted binaries of the image. For details, check the Tekton task log."
```

Comment:
the binaries comes from the image, I feel it is not image binaries, so can we say `extracted image binaries` here? @jseseCCS @dirgim

Reply:
@hongweiliu17 my proposal would be to just use `extracted binaries` since we've already implied that we've attempted to extract them from the image.

Reply:
oooh, i like "extracted binaries". simple!

so this? --> 

note="Task `$(context.task.name)` failed: Failed to extract image with `oc extract` command, so it cannot validate the extracted binaries. For details, check the Tekton task log."

but "...its extracted binaries." instead of "...the extracted binaries."? @hongweiliu17 @dirgim